src/main/resources/anniv6Resources/localization/eng/GremlinTown/Potionstrings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "${ModID}:LouseMilk": {
+    "NAME": "Louse Milk",
+    "DESCRIPTIONS": [
+      "Draw #b{0} cards and gain #b{1} Energy."
```

Comment:
Should Energy be yellow or capitalized?

Reply:
Capitalized, yes, yellow, no.  According to Energy Potion.